Code/GraphMol/MolInterchange/Parser.cpp

```diff
@@ -289,7 +579,7 @@ void readRDKitRepresentation(RWMol *mol, const rj::Value &repVal,
   if (!repVal.HasMember("formatVersion")) {
     throw FileParseException("Bad Format: missing format_version");
   }
-  if (repVal["formatVersion"].GetInt() > 1) {
+  if (repVal["formatVersion"].GetInt() > 2) {
```

Comment:
Should the version be in a define or constant variable?  It seems kind of oddly placed here.

Reply:
yep. good catch